cypress/integration/chi-vue/data-table.spec.js

```diff
@@ -111,10 +111,14 @@ describe('Data Table', () => {
   describe('Base', () => {
```

Comment:
Is the plan to create print-related E2E tests in a separate PR?
@jllr @evansito

Reply:
I don't mind either way